airflow/models/dag.py

```diff
@@ -2463,7 +2471,7 @@ def create_dagrun(
         conf: dict | None = None,
         run_type: DagRunType | None = None,
         session: Session = NEW_SESSION,
-        dag_hash: str | None = None,
+        dag_version_id: int | None = None,
```

Comment:
dag_version_id is a foreignkey from dag_version table to dagrun and auto increments based on how sqlalchemy works. However, it will eventually be refactored to be a UUID.

Reply:
> I am thinking about a backup/restore, how can we be sure durch such cases the IDs will not overwrite each other?

Since it's a foreign key, I don't see how it will overwrite. The above code is just for creating the dag version. Maybe we should change it to the dag_version object itself and not the ID?